### PR TITLE
Modify S5632: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5632/python/rule.adoc [hunk 1]
```diff
@@ -1,28 +1,32 @@
+This rule raises an issue when an object which doesn't derive from BaseException is raised.
+
 == Why is this an issue?
 
-In Python 3, attempting to raise an object which does not derive from BaseException will raise a ``++TypeError++``. In Python 2 it is possible to raise old-style classes but this shouldn't be done anymore in order to be compatible with Python 3.
+Attempting to raise an object which does not derive from BaseException will raise a ``++TypeError++``.
```
Comment (joke1196) [inline rules/S5632/python/rule.adoc:5]: BaseException here could be highlighted I think.

**Review by joke1196 — CHANGES_REQUESTED** [review]

**Comment by joke1196** [comment]
LGTM! Just added a small formatting comment.

File: rules/S5632/python/rule.adoc (hunk 1 above)
Comment (guillaume-dequenne-sonarsource) [reply rules/S5632/python/rule.adoc:5]: Good catch, thanks.

**Review by joke1196 — APPROVED** [review]
LGTM!

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**